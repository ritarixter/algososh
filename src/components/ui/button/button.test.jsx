import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from './button';


it('Кнопка с текстом рендерится без ошибок', () => {
  const tree = renderer
    .create(<Button text="Развернуть" />).toJSON();
    expect(tree).toMatchSnapshot();
}); 