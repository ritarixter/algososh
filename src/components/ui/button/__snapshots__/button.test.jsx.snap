// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Кнопка рендериться корректно Заблокированная кнопка рендерится без ошибок 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  disabled={true}
  type="button"
>
  <p
    className="text undefined"
  >
    Развернуть
  </p>
</button>
`;

exports[`Кнопка рендериться корректно Кнопка без текста рендерится без ошибок 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  />
</button>
`;

exports[`Кнопка рендериться корректно Кнопка с индикацией загрузки рендерится без ошибок 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  disabled={true}
  type="button"
>
  <p
    className="text undefined"
  >
    Развернуть
  </p>
</button>
`;

exports[`Кнопка рендериться корректно Кнопка с текстом рендерится без ошибок 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  >
    Развернуть
  </p>
</button>
`;
